chart/epinio/values.yaml

```diff
@@ -60,6 +60,12 @@ api:
       role: user
       workspaces:
         - workspace
+    - username: user1@epinio.io
+      passwordBcrypt: "$2a$10$6bCi5NMstMK781In7JGiL.B44pgoplUb330FQvm6mVXMppbXBPiXS"
+      role: user
+      workspaces:
+        - workspace
```

Comment:
Why do we need an additional user?

Reply:
I added it to test a user with an email. If you think it's not needed I can remove it. :)

Reply:
It would end up in everybody's cluster by default. If we need it in tests, we should create it in tests.